RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -10,17 +10,17 @@ Services
   }
   ReqProxy
   {
-    Port = 9198
+    Port = 9161
```

Comment:
We should be careful with changing the port numbers as some other VOs 
that are not LHCb may have already assigned 9161 to some other service.